companion/src/modeledit/logicalswitches.cpp

```diff
@@ -244,21 +244,12 @@ void LogicalSwitchesPanel::edited()
         updateTimerParam(cswitchOffset[i], model->logicalSw[i].val2, 0.1);
         break;
       case LS_FAMILY_EDGE:
-        cswitchOffset2[i]->setSpecialValueText(tr("(instant)"));
         if (sender() == cswitchOffset[i]) {
           model->logicalSw[i].val2 = TimToVal(cswitchOffset[i]->value());
-          updateTimerParam(cswitchOffset[i], model->logicalSw[i].val2, 0.0);
         }
         else {
           model->logicalSw[i].val3 = TimToVal(cswitchOffset2[i]->value()) - model->logicalSw[i].val2;
         }
-        updateTimerParam(cswitchOffset2[i], model->logicalSw[i].val2+model->logicalSw[i].val3, ValToTim(TimToVal(cswitchOffset[i]->value())-1));
-        if (model->logicalSw[i].val3 == 0) {
-          cswitchOffset2[i]->setSuffix("(infinite)");
-        }
-        else {
-          cswitchOffset2[i]->setSuffix("");
-        }
         break;
```

Comment:
These lines look like they are not needed, the same is happening in printLogicalSwitchLine() which is called from this function later.